transformer.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2017 ScyllaDB
+// Use of this source code is governed by a ALv2-style
+// license that can be found in the LICENSE file.
+
+package gocqlx
+
+import (
+	"reflect"
+
+	"github.com/gocql/gocql"
+)
+
+// bindTransformer is called right before binding a value to a named parameter.
+type bindTransformer func(name string, val interface{}) interface{}
+
+// UnsetEmpty unsets all empty parameters.
+// It helps to avoid tombstones when using the same insert/update
+// statement for filled and partially filled parameters.
+var UnsetEmpty = func(name string, val interface{}) interface{} {
```

Comment:
It should be more explicit that this is a transformer.

Reply:
Initially I though it would be some redundancy in word transformer but ok